docs/guide/lidar-sensors.md

```diff
@@ -171,19 +180,29 @@ SickLdMrs {
   SFVec3f    translation       0 0 0
   SFRotation rotation          0 1 0 0
   SFString   name              "Sick LD-MRS"
-  SFFloat    noise             0.3
-  SFInt32    numberOfLayers    4
-  SFFloat    angularResolution 0.008726646259972
+  SFString{"400001", "400102", "400001S01", "400102S01", "800001S01"}
```

Comment:
For consistency with the rest of the doc, I will keep the enum in the PROTO file, but document the enumerations in the field description rather than in this "code part".